src/components/tabs/js/tabsController.js

```diff
@@ -97,6 +99,92 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
     delete $attrs.$mdTabsTemplate;
   }
 
+  var DOMtabs = [];
+
+  function handleScroll () {
+    var nbTabs = ctrl.tabs.length;
+
+    var watcher = $scope.$watch(function () {
+      return $element.find('md-tab-content').length;
+    }, function (nv) {
+      if (nv === nbTabs) {
+        watcher();
+        refreshDOMtabs();
+      }
+    });
+  }
+
+  function refreshDOMtabs () {
+    DOMtabs = $element.find('md-tab-content');
+    if (ctrl.shrinkToolbar) {
+      bindScrollListener(ctrl.selectedIndex);
+    }
+  }
+
+  function bindScrollListener (newIndex, oldIndex) {
+    if (!DOMtabs.length) return;
+    if (oldIndex !== undefined && ctrl.tabs[oldIndex]) {
+      angular.element(DOMtabs[oldIndex]).off('scroll');
```

Comment:
You're unbinding all scroll listeners here, including anything that the user might have added.  Try unbinding only a method that you have a reference to.